server/utils/errors.ts

```diff
@@ -26,12 +26,3 @@ export class TaskListAPIError extends Error {
     this.field = field
   }
 }
-
-export class RestrictedPersonError extends Error {
```

Comment:
if the API is now returning a Forbidden error in these cases (LAO?), do we still want to handle that in a user friendly way in the UI?

Reply:
We are showing the error `You do not have permission to access this prison number` rather than `CRN: is restricted` which is both more accurate and gives less information about the ID that's been searched. I don't know if it's over cautious but my thinking is that the user should only know that they are not allowed to access that ID, not necessarily the reason why, because that would allow someone with malicious intent to get more info than they need.

Reply:
right yeah, sounds sensible 👍